RecoParticleFlow/PFRecHitProducer/interface/PFRecHitTopologyRecord.h

```diff
@@ -0,0 +1,17 @@
+#ifndef RecoParticleFlow_PFRecHitProducer_TopologyRecord_h
+#define RecoParticleFlow_PFRecHitProducer_TopologyRecord_h
+
+#include "FWCore/Framework/interface/EventSetupRecordImplementation.h"
+#include "FWCore/Framework/interface/DependentRecordImplementation.h"
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+#include "Geometry/Records/interface/HcalRecNumberingRecord.h"
+
+class PFRecHitHCALTopologyRecord : public edm::eventsetup::DependentRecordImplementation<
+                                       PFRecHitHCALTopologyRecord,
+                                       edm::mpl::Vector<HcalRecNumberingRecord, CaloGeometryRecord>> {};
+
+class PFRecHitECALTopologyRecord
+    : public edm::eventsetup::DependentRecordImplementation<PFRecHitECALTopologyRecord,
+                                                            edm::mpl::Vector<CaloGeometryRecord>> {};
```

Comment:
You could reuse the `CaloGeometryRecord` instead of defining a new record.
Unless you plan to add other dependencies in the future.